src/news/2015-11-16-phet.md

```diff
@@ -0,0 +1,27 @@
+---
+title: PhET Accessible Physics Simulations
+filename: phet
+date: '2015-11-16'
+---
+Floe is working with [PhET](http://phet.colorado.edu/)
+to improve the accessibility of their free, open, interactive physics simula
+tions. Over the past
```

Comment:
Please fix the line breaks in this article so no words are being split in half, it's adding many unnecessary spaces